Makefile

```
@@ -3,14 +1,27 @@
-	docker push soulgarden/swup:pass-0.0.10
+IMAGE ?= soulgarden/swup:pass-0.0.13
+PLATFORM ?= linux/amd64
+
+docker-build:
+	docker build . -t $(IMAGE) --platform $(PLATFORM)
+
+docker-push:
+	docker push $(IMAGE)
+
+# keep backward-compatible target that only builds (no push)
+build: docker-build docker-push
```

checkmake (0.2.2)

[warning] Target "build" should be declared PHONY.

(phonydeclared)